README.md

```diff
@@ -30,12 +30,22 @@ Base UI is an unstyled UI component library for building accessible user interf
 
 ---
 
+## Versions
+
+There are currently two Base UI libraries available on npm: [@base_ui/react](https://www.npmjs.com/package/@base_ui/react) (published from this repository)
+and [@mui/base](https://www.npmjs.com/package/@mui/base) (published from https://github.com/mui/material-ui/tree/master/packages/mui-base).
```

Comment:
There should probably be a note that the package name isn't final. I also think the note at the top could be better written to highlight the WIP status of this project and reduce confusion, and discourage using the old version